qa/common/infer_util.py

```diff
@@ -72,7 +80,8 @@ def infer_exact(tester, pf, tensor_shape, batch_size,
                 correlation_id=0, shm_region_names=None, precreated_shm_regions=None,
                 use_system_shared_memory=False, use_cuda_shared_memory=False,
                 priority=0, timeout_us=0):
-    tester.assertTrue(use_http or use_grpc or use_streaming)
+    tester.assertTrue(
+        use_http or use_http_json_tensors or use_grpc or use_streaming)
```

Comment:
GRPC has only one way of passing data to the server i.e. via our specified protobuf. HTTP on the other hand has two options i.e. binary or json. It would not make sense to have a common flag for both GRPC and HTTP.

Reply:
@tanmayv25 Is binary extension not supported in GRPC?

Reply:
For gRPC, the server can handle explicit tensors in the input, but in our client library we always use raw_contents to upload to the data to server.